ts/Dashboards/EditMode/SidebarPopup.ts

```diff
@@ -272,8 +274,12 @@ class SidebarPopup extends BaseForm {
         }
 
         // Remove highlight from the row.
-        if (editMode.editCellContext && editMode.editCellContext.row) {
-            editMode.editCellContext.row.setHighlight(true);
+        if (
+            editMode.editCellContext &&
+            editMode.editCellContext instanceof Cell &&
+            editMode.editCellContext.row
```

Comment:
`editMode.editCellContext instanceof Cell &&
            editMode.editCellContext.row` it should be enough.